lib/readthis/cache.rb

```diff
@@ -374,8 +374,14 @@ def exist?(key, options = {})
     #
     #   cache.clear #=> 'OK'
     #
-    def clear(_options = nil)
-      invoke(:clear, '*', &:flushdb)
+    def clear(options = nil)
```

Comment:
This can be changed to `options = {}`, and then you can simply do `if options[:async]` within the body.